docs/chapter_sorting/quick_sort.md

```diff
@@ -483,7 +483,7 @@ comments: true
     func medianThree(nums []int, left, mid, right int) int {
         if (nums[left] > nums[mid]) != (nums[left] > nums[right]) {
             return left
-        } else if (nums[mid] < nums[left]) != (nums[mid] > nums[right]) {
+        } else if (nums[mid] > nums[left]) != (nums[mid] > nums[right]) {
```

Comment:
It seems that this problem is also found in other code, you can modify the codes meanwhile.

Reply:
> It seems that this problem is also found in other code, you can modify the codes meanwhile.

No need to fix other codes.